_content/news/_posts/2019-02-16-valentijn.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Valentijnsmatch"
+date:   2019-01-16 19:00:00
+description: Ter ere van valentijn geven wij zaterdag een sauna arrangement weg.
+cover: /news/img/saunakarmijn.jpg
+permalink: /news/2019-02-16-valentijn/
+---
+
+Ter ere van Valentijn, geven we zaterdag 16 februari, tijdens de wedstrijd van onze Dames A tegen de Antwerp Giants, een sauna arrangement weg! Dit arrangement wordt geschonken door [privé sauna en wellness Karmijn](https://www.saunakarmijn.be/).
```

Comment:
Tijdens de wedstrijd van onze Dames A tegen Antwerp Giants geven we een sauna-arrangement weg, ter gelegenheid van Valentijn. Dit arrangement wordt geschonken door [privé sauna en wellness Karmijn](https://www.saunakarmijn.be/).